public/app/features/dashboard/components/GenAI/hooks.ts

```diff
@@ -12,14 +12,20 @@ import { isLLMPluginEnabled, OPEN_AI_MODEL } from './utils';
 // Ideally we will want to move the hook itself to a different scope later.
 type Message = openai.Message;
 
+export enum StreamStatus {
+  IDLE = 'idle',
```

Comment:
For non-English speakers, `idle` might be confusing 😅 What do you think about `non-started`? Do you think it is accurate?

Reply:
`inactive`?

Reply:
Actually, after checking multiple APIs, idle seems to be the standard :)